pdr_backend/analytics/predictoor_dashboard/dash_components/callbacks.py

```diff
@@ -142,13 +145,49 @@ def update_predictoors_table_on_search(
                 predictoors_data, "user", search_value, selected_predictoors
             )
 
+        if user_payout_stats:
+            # Her bir data elemanı için, eşleşen user_payout_stat bulunur ve güncellenir.
+            # Eşleşme yoksa, orijinal data elemanı korunur.
```

Comment:
I think we should translate this to english 😄

Reply:
Sorry my fault :D